customer/appsettings.Development.json

```diff
@@ -7,8 +7,8 @@
     }
   },
   "APIs": {
-    "GuestExperience": "http://localhost:8081/",
-    "TableService": "http://localhost:8082/",
-    "Billing": "http://localhost:8083/"
+    "GuestExperience": "http://GuestExperience:8080/",
+    "TableService": "http://TableService:8080/",
+    "Billing": "http://Billing:8080/"
```

Comment:
Can you create an additional configuration file for the Docker-hosted configuration?
So we have
- `Development` targeting `localhost` and
- e.g. `DevelopmentDocker` targeting a docker compose setting?

Or the other way around - I don't have a strong opinion :-) I think it makes sense to separate these two modes a bit for development purposes.